### ISSUE TITLE
Is a PM allowed to see details for each project?

### ISSUE THREAD

**Issue by at-midorinezu** [AsianTechInc/AT-Portal#16]
•Currently, PM cannot see project details: who are in a project.
•PM cannot set an officer.

![screen shot 2016-11-16 at 15 26 58](https://cloud.githubusercontent.com/assets/19849446/20339910/75980ad8-ac11-11e6-9162-110ab41b985c.png)

**Comment by at-datnm** [comment]
@at-huydq : please answer this issue

**Comment by at-huydq** [comment]
@at-midorinezu Yeah I can imagine you hovering over the project name to see a project card showing most of the basic information about the project. Definitely something good to have in the future. 

As for the 2nd question I don't understand, can you raise it in a separate issue please.

**Comment by at-huydq** [comment]
@at-midorinezu data ACL issue, can you access the project now?

**Comment by at-midorinezu** [comment]
Thank you for the answer, and sorry for the reply.
I could access to the project details.

Thank you.
